### ISSUE TITLE
webp background

### ISSUE THREAD

**Issue by fourroses666** [mplavala/webpconverter#11]
I'm not sure why but when I look at the webpbackground snippet I see 2x 
<php
<php

Also the info is not added while it looks like its there..
https://github.com/mplavala/webpconverter/blob/master/install/assets/snippets/webpBackground.tpl

I would prefer to have added this below  info to the background snippet so I don't have to look how to do it and can look into the snippet:
```// style="background-image:url([!webpBackground? &src=`[[phpthumb? &input=`[+img+]` &options=`w=600`]]`!])"```

**Comment by mplavala** [comment]
By adding info you mean to add use case to the snippet as a comment inside the code?

**Comment by fourroses666** [comment]
Yes, or at least add the Github link. 

Something like:

```
/**
 * webpBackground
 * Evolution CMS plugin and snippet to convert all images on the page to WebP format.
 * @category    snippet
 * @reportissues https://github.com/mplavala/webpconverter/issues/
 * @documentation Github repositorie https://github.com/mplavala/webpconverter
 */
 ```

**Comment by mplavala** [comment]
All fixed in commit d321321d6a0698e5b8717efb97be5a636f2670f7.
